docker/bam-readcount/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM seqfu/libnss:latest
```

Comment:
Is this a stable base image, or should we use a base of ubuntu or something and include whatever is in your libnss image.  (open to either - just a thought)

Reply:
I think ubuntu is better

Reply:
I'm actually inclined to remove the Docker directory. These were really to ease development for me and I think they make the `BUILD.md` documentation confusing. But if we do keep them, definitely agree Ubuntu.

Reply:
Removed for now, retained in [docker-dev](https://github.com/seqfu/bam-readcount/tree/docker-dev) branch.